projects/RabbitMQ.Client/client/api/BasicProperties.cs

```diff
@@ -85,7 +85,7 @@ public BasicProperties()
         {
         }
 
-        public BasicProperties(ReadOnlyBasicProperties input)
```

Comment:
Is this change related to NRT? If not, consider to extract this breaking change into a separated PR.

Reply:
Yes, it appears that this change is necessary due to it's use in `PopulateActivityAndPropagateTraceId`

Reply:
AFAIK the `ReadOnlyBasicProperties` is anyway new in 7.0, so this should not be problematic